setup.py

```diff
@@ -7,7 +7,9 @@
 
 dependencies = [
     "clvm>=0.9.2",
-    "clvm_tools_rs>=0.1.37"
+    "clvm_tools_rs>=0.1.37",
+    "importlib_metadata~=6.11.0",
+    "setuptools",
```

Comment:
Since you updated away from `pkg_resources`, is this `setuptools` dependency still needed?  Maybe we still use `pkg_resources` elsewhere.

Reply:
setuptools is needed for distutils (used for log and Command)